### PR TITLE
mariadb chart using bitnami/mariadb image

### PR DESCRIPTION
**Pull request by sameersbn**
- Use's the [bitnami/mariadb](https://github.com/bitnami/bitnami-docker-mariadb) docker image 
- Will be actively maintained by [Bitnami](https://bitnami.com)

### CODE REVIEW

File: mariadb/README.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+```bash
+$ helm install mariadb-x.x.x.tgz
+```
+
+## Introduction
+
+This chart bootstraps a [MariaDB](https://github.com/bitnami/bitnami-docker-mariadb) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Get this chart
```
Comment (prydonius) [inline mariadb/README.md:17]: This section will need to be updated once we have the Chart Repo going

Comment (sameersbn) [inline mariadb/README.md:17]: Right. Sorry for not pointing it out in the PR

File: mariadb/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Name }}"
```
Comment (michelleN) [inline mariadb/templates/deployment.yaml]: @sameersbn did you mean for this to be "{{ .Chart.Name }}-{{ .Chart.Version }}" ?

Comment (sameersbn) [inline mariadb/templates/deployment.yaml]: oops.. will fix it. thanks @michelleN

Comment (sameersbn) [inline mariadb/templates/deployment.yaml]: updated!

**Pull request merged**